common/openapi.yml

```diff
@@ -158,6 +158,23 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/batch/v1/runs/{runId}/state:
+    parameters:
+      - $ref: '#/components/parameters/RunId'
+    patch:
```

Comment:
Will Cromwell be sending a patch or a post?

Reply:
Post it is.